requirements/common.txt

```diff
@@ -4,14 +4,12 @@ boto3==1.17.36
 cachetools==5.2.0
 dacite==1.6.0
 Deprecated==1.2.11
-eventlet==0.31.0
 Flask==1.1.2
 ftfy==6.1.1
 httpx==0.23.0
 hypercorn==0.13.2
 importlib-metadata==1.4.0; python_version<"3.8"
 Jinja2==3.0.3
-kaleido==0.2.1
```

Comment:
`kaleido` is required to export plotly plots as static images:
https://github.com/voxel51/fiftyone/blob/3c0cd9d9ab596ad18506bfeb02a3328638eb8832/fiftyone/core/plots/plotly.py#L1583

Reference
https://plotly.com/python/static-image-export/